.eslint-resolver.js

```diff
@@ -0,0 +1,22 @@
+const path = require('path');
+const resolve = require('resolve')
+
+exports.interfaceVersion = 2
+
+const dotJS = /\.js$/;
+exports.resolve = function (source, file, config) {
+  if (resolve.isCore(source)) return { found: true, path: null }
+
+  source = source.replace(dotJS, '.ts');
```

Comment:
```suggestion
  source = source.replace(/\.js$/, '.ts');
```
nit: seems like we could just inline the `dotJS` string here